src/jemalloc.c

```diff
@@ -37,6 +37,13 @@ bool	opt_abort =
 #endif
     ;
 bool	opt_abort_conf =
+#ifdef JEMALLOC_DEBUG
```

Comment:
We don't really want to output conf by default in debug mode; we have many testing frameworks running in dbg mode, with an expected test output format.